src/dataService/tagDataService.ts

```diff
@@ -123,8 +125,10 @@ export class TagDataService {
       .whereNotIn('tag', existingTags)
       .andWhere({ 'item_tags.user_id': parseInt(this.userId) })
       //for now, considering time_updated from list table when item_tags
-      //time_updated is null.
-      .orderByRaw('COALESCE(item_tags.time_updated, list.time_updated) DESC')
+      // time_added is null.
+      // Not using item_tags.time_updated or coalescing with item_tags.time_added
```

Comment:
extreme nit but just clarifying: `Not using item_tags.time_updated . Instead, coalescing with item_tags.time_added`

Reply:
This was incredibly unclear language. I can update. I'm actually saying:

Not using or coalescing the item_tag.time_updated field because it's so unreliable and not worth it